### PR TITLE
Copy-edit-repo-project-tutorial-part2

### LINKED ISSUE #357: https://github.com/Umuzi-org/ACN-syllabus/tree/develop/content/projects/tilde/repo-card-tutorial/part-2/_index.md

### PR DESCRIPTION
**Pull request by asradebe**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [ ] I ran the hugo server and looked at my changed in the browser with my own eyes
- [ ] I ran the linter and there were no errors
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

**Review by JacobUmuzi — CHANGES_REQUESTED** [review]
line 32 (new) maintain previous sentence, the word 'will' sounds a bit assertive on the familiarity. 'to complete is also correct as it refers to the complete column.

**Comment by asradebe** [comment]
> line 32 (new) maintain previous sentence, the word 'will' sounds a bit assertive on the familiarity. 'to complete is also correct as it refers to the complete column.
Jacob those little changes were suggested by Grammarly. If we change it back to its original state then nothing will be changed on that card. I believe it should be good as it is. I used Sheena's styling guide to also help me. Can we maybe get a second opinion?

File: content/projects/tilde/repo-card-tutorial/part-2/_index.md [hunk 1]
```diff
@@ -12,23 +12,21 @@ submission_type: continue_repo
...
 
 - Make a new branch
-- Add a bit more text to your markdown file.
-- Make a new PR
+- Edit and add a bit more text to the markdown file you created previously.
+- Make a new Pull Request
 - Get it merged
 
-## Move your card to the REVIEW column (AFTER YOUR PR IS MERGED! NOT BEFORE!)
+## Move your card to the REVIEW column (AFTER YOUR PULL REQUEST IS MERGED! NOT BEFORE!)
```
Comment (ry-oc) [inline content/projects/tilde/repo-card-tutorial/part-2/_index.md:30]: this is a comment

File: content/projects/tilde/repo-card-tutorial/part-2/_index.md [hunk 2]
```diff
@@ -12,23 +12,21 @@ submission_type: continue_repo
...
 
-This should feel very familiar. Do what it takes to get your card to complete.
```
Comment (ry-oc) [inline content/projects/tilde/repo-card-tutorial/part-2/_index.md:34]: this is acomment

**Review by ry-oc — CHANGES_REQUESTED** [review]

**Review by JacobUmuzi — APPROVED** [review]

**Pull request merged**